Update map template url parameters (#124)
MapsPeople/docsite: docs/10-Map Templates/web-map-template.mdx

### Supported properties on the MapsIndoors Map component​

> The MapsIndoors Map component supports a list of props (see the list below) that can be configured in the `App.jsx` file. 
> When loading the MapsIndoors Map component for the first time, the map will respect the default values set for the `apiKey`, `venue`, `logo` and `primaryColor` props, which can be found in the `MapsIndoorsMap.jsx` file under the `defaultProps` object. 

Reviewer: In Markdown, you have to have an empty line between two paragraphs, or the linter will complain about it not being valid.

```suggestion
The MapsIndoors Map component supports a list of props (see the list below) that can be configured in the `App.jsx` file.

When loading the MapsIndoors Map component for the first time, the map will respect the default values set for the `apiKey`, `venue`, `logo` and `primaryColor` props, which can be found in the `MapsIndoorsMap.jsx` file under the `defaultProps` object. 
```

Add attributes to the web component as needed (see supported properties below).

> **Note!** The `external-IDs` and `app-user-roles` expect an array, which in a web component is handled differently (see example below).
> In your script, define the array of external IDs that you want to be shown on the Map Template. 
> Then get a hold of the mapsIndoors map element using the `document.querySelector()` method. 
> When you have the mapsIndoors map element, assign its prop `externalIDs` the array of external IDs that you defined at the beginning.

Reviewer: ```suggestion
**Note!** The `external-IDs` and `app-user-roles` expect an array, which in a web component is handled differently (see example below).

In your script, define the array of external IDs that you want to be shown on the Map Template. Then get a hold of the `mapsIndoorsMapElement` using the `document.querySelector()` method. When you have the `mapsIndoorsMapElement`, assign its prop `externalIDs` the array of External IDs that you defined at the beginning.
```

```javascript
const externalIDsArray = ["externalID-1", "externalID-2"]
const mapsIndoorsMapElement = document.querySelector('mapsindoors-map')
mapsIndoorsMapElement.externalIDs = externalIDsArray;
```

> Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.
> 
> 
> ### Using just the browser

Reviewer: ```suggestion
Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.

### Using just the browser
```

```

> Add attributes to the web component as needed (see supported properties above).
> Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.

Reviewer: ```suggestion
Add attributes to the web component as needed (see supported properties above).

Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.
```

```

> Add properties to the MapsIndoorsMap component as needed (see list above).
> Use query parameters to configure the MapsIndoorsMap component by setting the `supportsUrlParameters` prop to `true`.

Reviewer: ```suggestion
Add properties to the MapsIndoorsMap component as needed (see list above).

Use query parameters to configure the MapsIndoorsMap component by setting the `supportsUrlParameters` prop to `true`.
```

## Query Parameters

> The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component if the `supportsUrlParameters` property is set to true.
> In addition, the Web component and the React component also support using the URL parameters if the `supports-url-parameters` attribute respectively the `supportsUrlParameters` prop is set to `true`.

Reviewer: ```suggestion
The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component if the `supportsUrlParameters` property is set to true.

In addition, the Web component and the React component also support using the URL parameters if the `supports-url-parameters` attribute or the `supportsUrlParameters` prop is set to `true`.
```
